MoreLinq/Sequence.cs

```diff
@@ -17,31 +17,31 @@
 
 namespace MoreLinq
 {
-    using System;
     using System.Collections.Generic;
-    using System.Linq;
 
     static partial class MoreEnumerable
     {
         /// <summary>
-        /// Generates a sequence of integral numbers within the specified range.
+        /// Generates a sequence of integral numbers within the (inclusive) specified range.
+        /// If sequence is ascending the step is +1, otherwise -1.
         /// </summary>
         /// <param name="start">The value of the first integer in the sequence.</param>
         /// <param name="stop">The value of the last integer in the sequence.</param>
         /// <returns>An <see cref="IEnumerable{Int32}"/> that contains a range of sequential integral numbers.</returns>
         /// <example>
         /// <code>
-        /// string result = Sequence(6, 0);
+        /// var result = MoreEnumerable.Sequence(6, 0);
         /// </code>
         /// The <c>result</c> variable will contain <c>{ 6, 5, 4, 3, 2, 1, 0 }</c>.
         /// </example>
+
         public static IEnumerable<int> Sequence(int start, int stop)
         {
             return Sequence(start, stop, start < stop ? 1 : -1);
         }
 
         /// <summary>
-        /// Generates a sequence of integral numbers within the specified range.
+        /// Generates a sequence of integral numbers within the (inclusive) specified range.
```

Comment:
No mention of `step`? Something along the lines of:

> An additional parameter specifies the steps in which the integers of the sequence increase or decrease.

Worth also adding a [`<remarks>`](https://msdn.microsoft.com/en-us/library/3zw4z1ys.aspx) section about the execution semantics and a warning about what happens when `step` is zero (sequence will be `start` repeating forever).